src/App/App.css

```diff
@@ -0,0 +1,23 @@
+.button {
+  background-color: #787978; /* Green */
```

Comment:
why would you use scss in the project? It's more maintainable and readable

Reply:
fixed